tests/acceptance/page_objects/base.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import
+
+
+class BasePage(object):
```

Comment:
I got a BasePage class merged into master in `acceptance/pages.py` do you want a hand in joining it with your implementation as having two seems redundant.